src/components/AddressSearch.js

```diff
@@ -111,11 +138,28 @@ const AddressSearch = (props) => {
             }}
             textInputProps={{
                 InputComp: TextInput,
+                ref: (node) => {
+                    if (!props.innerRef) {
+                        return;
+                    }
+
+                    if (_.isFunction(props.innerRef)) {
+                        props.innerRef(node);
+                        return;
+                    }
+
+                    // eslint-disable-next-line no-param-reassign
+                    props.innerRef.current = node;
+                },
                 label: props.label,
                 containerStyles: props.containerStyles,
                 errorText: props.errorText,
                 value: props.value,
-                onChangeText: (text) => {
+                isFormInput: props.isFormInput,
+                inputID: props.inputID,
+                shouldSaveDraft: props.shouldSaveDraft,
+                onBlur: props.onBlur,
+                onChange: (text) => {
```

Comment:
Maybe I'm wrong, but I believe that `onChanteText` passes the text value as argument vs `onChange` passes an event. We should make sure that we are passing the text value here.

Reply:
I think it returns only text value, considering `AddressSearch` is using `TextInput` as the base component.

https://github.com/Expensify/App/blob/c049cc4d16ec71606fb635a7d7913d21b0e876f1/src/components/TextInput/BaseTextInput.js#L240
https://github.com/Expensify/App/blob/c049cc4d16ec71606fb635a7d7913d21b0e876f1/src/components/TextInput/BaseTextInput.js#L121-L128

Reply:
I don't think so.
Should I push this diff?
```diff
diff --git a/src/components/AddressSearch.js b/src/components/AddressSearch.js
index 005438f75..bee2e40cb 100644
--- a/src/components/AddressSearch.js
+++ b/src/components/AddressSearch.js
@@ -158,7 +158,7 @@ const AddressSearch = (props) => {
                 inputID: props.inputID,
                 shouldSaveDraft: props.shouldSaveDraft,
                 onBlur: props.onBlur,
-                onChange: (text) => {
+                onChangeText: (text) => {
                     if (skippedFirstOnChangeTextRef.current) {
                         props.onChange({street: text});
                     } else {
```

Reply:
@kakajann Sorry for the late reply, I was ooo for most of last week. I think it's fine to keep `onChangeText` as we had before. The `onChange` handler will be passed by the parent Form component.

Reply:
@luacmartins, done